### PR TITLE
[flutter_markdown] Standardize license/copyright

### PR DESCRIPTION
**Pull request by stuartmorgan**
This applies the recent license standardization to the newly-imported
flutter_markdown to align it with the rest of the respository:
- Standardizes year as 2013
- Standardizes author as "The Flutter Authors", moving other authors
  to the AUTHORS file
- Uses the exact same text format for the copyright line as in other
  files

Note that the files that currently list "Quiverware" are not third-party
code, but code that was submitted to flutter/flutter_markdown through
the standard PR process with a signed CLA; the reviews just didn't catch
that it was using a non-standard author line.

## Pre-launch Checklist

- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy].
- [ ] I updated `CHANGELOG.md` to add a description of the change.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

**Comment by stuartmorgan** [comment]
Since I was doing the repo import and the copyright standardization in parallel, I forgot to apply the latter to the former; ideally this would have been part of the initial import (and in the future that will be enforced once I finish the cleanup and enable the enforcement via CI).

**Comment by Hixie** [comment]
I do not believe we can make the changes described in this PR. Let's chat offline.

**Comment by stuartmorgan** [comment]
@mjordan56 As the original author of the PRs that added the files with these copyrights, would you be willing to leave a comment indicating that you approve of this change?

The context here is that our goal in Flutter repositories is to use a single, standard copyright line for any code that is not third-party code. Ideally this would have been flagged during the initial review, but the flutter_markdown repository didn't have any tooling to catch it so it wasn't noticed. We'd like to retroactively fix it to match other code that was submitted under the CLA, but would like your approval in doing so. As you can see in the PR, "Quiverware LLC" will continue to be credited, just via the AUTHORS file.

**Review by mjordan56 — APPROVED** [review]
@stuartmorgan Thank you for the heads up on this change. Everything looks fine to me. I made one comment with respect to the addition of Quiverware LLC to the AUTHORS document. You can add an email address for Quiverware LLC if you like but I'm also fine with the change as is.

File: packages/flutter_markdown/AUTHORS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Below is a list of people and organizations that have contributed
+# to the Flutter project. Names should be added to the list like so:
+#
+#   Name/Organization <email address>
+
+Google Inc.
+The Chromium Authors
+Quiverware LLC
```
[the remark this answers is not in the document]
Comment (stuartmorgan) [reply packages/flutter_markdown/AUTHORS]: Thanks, added!

**Review by Hixie — APPROVED** [review]
Thanks @mjordan56!

LGTM

**Comment by stuartmorgan** [comment]
I'm not sure why it's saying submit-queue is red; the last run is green. Landing.

**Pull request merged**